packages/core/src/CenteredContent/CenteredContent.js

```diff
@@ -12,24 +12,31 @@ import propTypes from '@dhis2/prop-types'
  */
 const CenteredContent = forwardRef(
     ({ className, dataTest, children, position }, ref) => (
-        <div className={cx(className, position)} data-test={dataTest} ref={ref}>
-            {children}
+        <div
+            className={cx('centered-content', className, position)}
+            data-test={dataTest}
+            ref={ref}
+        >
+            <div className="centered-inner-content">{children}</div>
+
             <style jsx>{`
-                div {
-                    position: absolute;
-                    left: 50%;
+                .centered-content {
+                    width: 100%;
+                    height: 100%;
```

Comment:
Dimensions will be relative to the parent element, whereas previously, the dimensions were relative to the first positioned element. That could break layouts that rely on this.

Reply:
Your observation is correct. I'd say the previous behaviour can be considered a bug. In practice, this component is mostly used in combination with a layer, or a component-cover, so in most cases the UI _should_ remain unaffected.

Reply:
Hey @HendrikThePendric @ismay, I'm not sure I understand... is the "breaking change" here the fact that there is an additional `<div>` in the DOM tree?  How are `Dimensions` relevant here?

Reply:
In any case, since the previous behavior wasn’t documented and caused issues I think labeling it a fix is fine, so long as we do some spot checks on places using it and document the new intentional behavior

Reply:
@amcgee and I discussed this on Slack using [this example](https://codesandbox.io/s/icy-surf-7nepe?file=/index.html) and decided we can classify this as a bugfix.